documentation/decisions/adr/2023-12-15-deliverable-reporting-strategy.md

```diff
@@ -0,0 +1,235 @@
+
+### Limiting the 30k deliverables included in reporting
+
+- **Label:** Use a label to indicate when a deliverable should be included in reporting
+- **Status:** Use the status of the deliverable in the provisional roadmap project
+- **GitHub action:** Explicitly list the deliverables to include in the GitHub action for reporting
+
+## Decision outcomes <!-- REQUIRED -->
+
+### Assigning issues to deliverables
+
+We've decided to use a "deliverable" column to assign issues to a parent 30k deliverable. This involves:
+
+- creating a single select column in both the product roadmap and the sprint board
+- populating that column with options that represent each 30k deliverables in our roadmap
+- using that column to indicate the 30k deliverable that an issue is assigned to
+
+- **Positive outcomes**
+  - We can filter, group, and sort issues by deliverable in GitHub projects and GitHub's automated insight reporting.
+  - The engineering team can continue to use GitHub milestones to organize issues into units of work that are smaller than a 30k deliverable.
+- **Negative outcomes**
+  - We won't be able to filter for issues assigned to a given 30k deliverable within the GitHub repository.
+  - We'll have to make sure the list of options in the deliverable column is consistent across GitHub projects in order to join issues from different projects correctly in our custom reporting.
+  - We'll have to update the current logic in our custom reporting so that issues are joined to their parent deliverable using the value of this column.
+
+> [!NOTE]
+> If the team finds a consistent need to filter the list of issues by deliverable within the repository, we may revisit this decision and choose to use ***both*** a label and a deliverable column to assign an issue to a 30k deliverable. We've decided **not** to use both options for the time being to avoid having multiple (potentially conflicting) ways of assigning an issue to a given deliverable.
+
+### Limiting the 30k deliverables included in reporting
+
+We've decided to use the status of the deliverable in the product roadmap board to determine when deliverables are included or excluded from reporting.
+
+- **Positive outcomes**
+  - Keeps our reporting strategy closed aligned with how we're monitoring our delivery progress.
+  - Avoids creating *another* field or label that needs to be maintained separately.
+- **Negative outcomes**
+  - Tightly couples reporting logic with delivery management, which may introduce challenges if there are instances in which we want to report on some but not all deliverables in a given status.
+  - If stakeholders aren't familiar with the relationship between deliverable status and reporting, it may be harder for them to understand why certain deliverables appear in a given report but others do not.
+
+## Evaluation - Assigning issues to deliverables <!-- OPTIONAL -->
+
+### Milestone
```

Comment:
Hmm, it seems like there's actually two different ways to organize deliverables by milestone, but we've only evaluated one here. 

Milestone Organization A: what you've described (one milestone per 30k) 

Milestone Organization B: what we currently do (each 30k can have multiple milestones which all "roll up" into the 30k) 

Should we evaluate option B as well? Option B would, like some of the other options, "Allow the engineering team to continue to use GitHub milestones to organize issues into units of work that make sense to them."

Reply:
Good point, I added a section for our current strategy in this commit: [feat: Adds option for creating milestones per 10k](https://github.com/HHS/simpler-grants-gov/pull/926/commits/37e3a1186457130b19343274391c96ce5b2ab1f7)